about.html

```diff
@@ -0,0 +1,110 @@
+                        development, though activities of Devcon continue year-round</span></p>
+            </div>
+            <p id="contact">Please contact us per Email for any further questions about Gamecon 2023<br><a
+                    href="https://www.gamimgfair2023@gamecon.org" type="Email">gamimgfair2023@gamecon.org</a></p>
+        </section>
+
+        <section id="logoDiv">
+            <h2 class="titles">Gaming Trade Fair,<br class="mobbr">Gamecon 2023 Logo</h2>
+            <i class="aboutdash"></i>
+            <p id="logoDesc">The logo of Gamecon 2023 was decided through the logo competition <span class="desbr">After
+                    online public voting and Selection committee's judging process, the logo from Soban Syeed won the
+                    prize. After consultation with Mr. Syeed, we optimized the prize-winning logo for this year's
+                    Gamecon as following:</span></p>
+            <div id="logoBox">
+                <img src="./images/Gamecon2023.png" alt="Gamecon Logo" id="logo">
+            </div>
+            <i class="hrGrey"></i>
+        </section>
+
+        <section id="history">
+            <h2 class="titles">See the previous Gamecon</h2>
+            <i class="aboutdash"></i>
+            <p id="historyDesc">Take a look at last two Gamecons</p>
+            <div id="oldConContainer">
+                <div class="oldGamecon" id="g22">
+                    <h2 class="previousYear">2022<br>
+                        <p class="previousDetail">Gamecon 2022 in Germany</p>
+                    </h2>
+                </div>
+                <div class="oldGamecon" id="g21">
+                    <h2 class="previousYear">2021<p class="previousDetail">Gamecon 2021 in Germany</p>
+                    </h2>
+                </div>
+            </div>
+        </section>
+
+        <section id="sponser">
+            <h2 id="sponserTitle">Sponsers</h2>
+            <i class="dash guestDash"></i>
+            <div id="sponserList">
+                <img src="./images/pngkey.com-cigna-logo-png-8644440.png" alt="Cigna" class="sponsers">
+                <img src="./images/pngkey.com-games-png-1127920.png" alt="Qubo2" class="sponsers">
+                <img src="./images/pngkey.com-northrop-grumman-logo-png-10087161.png" alt="Walmart" class="sponsers">
+                <img src="./images/pngkey.com-patriot-logo-png-5303036.png" alt="Patriot" class="sponsers">
+                <img src="./images/pngkey.com-zico-png-3762944.png" alt="Zico" class="sponsers">
+            </div>
+        </section>
+    </main>
+
+    <footer>
+        <h2 id="footerHead">&COPY; Gamecon</h2>
+        <p id="copyright">2023 Gamecon. <br class="mobbr">All Rights Reserved<span class="desbr"><br>Lorem ipsum dolor
+                sit
+                amet consectetur adipisicing elit.</span></p>
+    </footer>
```

Comment:
- [ ] As per design requirements, kindly make sure that your desktop's `About` page footer is of a dark color background, refer back to the design to extract the needed color.

Reply:
Thanks for the review. I have changed it as per the template

![Screenshot 2023-06-30 013156](https://github.com/SnakyMz/Gamecon/assets/115364362/699cb78a-d0c4-47bd-a439-3ac5ad41351f)